packages/usa/vtex.country-data-settings/configuration.json

```diff
@@ -232,11 +232,11 @@
       "maxLength": 100
     },
     "street": {
-      "label": "street",
+      "label": "streetRoadAvenue",
```

Comment:
why update the label only for USA? wouldn't it make more sense to only update the translation in place components for english? because this way it will generate a new i18n key that will probably be "translated" to other languages even though it is only used in english

Reply:
It's not generating a new key, USA has wrong keys for this fields, [other languages uses this new value](https://github.com/vtex-apps/countries-data/blob/main/packages/bra/vtex.country-data-settings/configuration.json#L136)

Reply:
ohhh my bad then